.coveragerc

```diff
@@ -6,3 +6,20 @@ omit =
 
 [report]
 include = piptools/*, tests/*
+
+# Regexes for lines to exclude from consideration
+exclude_lines =
```

Comment:
I don't think we should add this configuration. IMO, it is okay to understand that coverage isn't 100% and doing so paints a more accurate picture about what the coverage numbers mean. Plus, in this case, I think we could add a test.